### PR TITLE
[Console] Do not duplicate Helper::strlen() code

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Console/Application.php [hunk 1]
```diff
@@ -1086,19 +1087,6 @@ public function setDefaultCommand($commandName)
         $this->defaultCommand = $commandName;
     }
 
-    private function stringWidth($string)
-    {
-        if (!function_exists('mb_strwidth')) {
```
Comment (ro0NL) [inline src/Symfony/Component/Console/Application.php]: Should we add this to `Helper::strlen` as well?

Comment (ogizanagi) [inline src/Symfony/Component/Console/Application.php]: It already did: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Helper/Helper.php#L52-L63

The two methods were strictly the same (and changes in upper branches were reported on both identically)

Comment (ro0NL) [inline src/Symfony/Component/Console/Application.php]: Oops.. checked master :) :+1:

**Comment by fabpot** [comment]
Good catch, thanks @ogizanagi.

**Pull request merged**